src/main/java/kr/codesqaud/cafe/controller/ArticleController.java

```diff
@@ -0,0 +1,54 @@
+package kr.codesqaud.cafe.controller;
+
+import kr.codesqaud.cafe.domain.Article;
+import kr.codesqaud.cafe.repository.ArticleRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@Controller
+public class ArticleController {
+
+    private ArticleRepository articleRepository;
+
+    Logger logger = LoggerFactory.getLogger(getClass());
+
+    public ArticleController(ArticleRepository articleRepository) {
+        this.articleRepository = articleRepository;
+        logger.debug("articleController");
+    }
+
+    @PostMapping("/questions")
+    public String addContents(@RequestParam String title, @RequestParam String writer, @RequestParam String contents) {
```

Comment:
```suggestion
    public String save(@RequestParam String title, @RequestParam String writer, @RequestParam String contents) {
```

userRepo 가 save이니 일관성을 위해서 여기도..